test/fuzz/gen.ml

```diff
@@ -36,17 +37,22 @@ let expr_available_1_i32 if_else expr ~locals ~stack env =
     ]
   in
   [ pair B.iunop_32 (const [ S.Nothing ])
+  ; if_else expr ~locals ~stack env
   ; pair B.itestop_32 (const [ S.Nothing ])
-  ; pair B.i64_extend_i32 (const [ S.Pop; S.Push (Num_type I64) ])
-  ; pair B.extend_32_i32 (const [ S.Nothing ])
   ; pair B.f32_convert_i32 (const [ S.Pop; S.Push (Num_type F32) ])
   ; pair B.f64_convert_i32 (const [ S.Pop; S.Push (Num_type F64) ])
   ; pair B.f32_reinterpret_i32 (const [ S.Pop; S.Push (Num_type F32) ])
-  ; if_else expr ~locals ~stack env
   ]
+  @ ( if env.Env.conf = Concrete then
+        [ pair B.extend_32_i32 (const [ S.Nothing ])
+        ; pair B.i64_extend_i32 (const [ S.Pop; S.Push (Num_type I64) ])
+        ; pair B.iunop_popcnt_32 (const [ S.Nothing ])
+        ]
+      else [] )
   @ B.local_set_i32 env @ B.local_tee_i32 env @ B.global_set_i32 env
-  @ (if B.memory_exists env then B.memory_grow :: load_instr else [])
-  @ B.expr_br_if env stack @ B.table_get env
+  @
+  if B.memory_exists env then B.memory_grow :: load_instr
+  else [] @ B.expr_br_if env stack @ B.table_get env
```

Comment:
I believe this is wrong (or confusing) due to wrong parenthesis, it should be:

```ocaml
( if B.memory_exists env then B.memory_grow :: load_instr
  else [] )
@ B.expr_br_if env stack @ B.table_get env
```

Reply:
Parenthesis like this: that's always been my intention ;)